### DISCUSSION TITLE
tls: failed to verify certificate: x509: certificate signed by unknown authority

### DISCUSSION THREAD

**Discussion by melwin145** [jfrog/jfrog-cli#2112] [Q&A]
`jf config add `

Facing tls: failed to verify certificate: x509: certificate signed by unknown authority for windows 10 PC 
how to resolve it?

We are using the intranet only. self-signed certification used for internal access. working fine with firefox and edge

**Comment by ra-bart** [comment]
For me, [exporting the site certificate](https://superuser.com/a/1292098) and placing it in the below directory got **jf** working with our Artifactory;

`
PS C:\> mkdir $HOME\.jfrog\security\certs
`
